CHANGES/38.bugfix

```diff
@@ -0,0 +1 @@
+Fix '--signing-service' option not allowing HREF
```

Comment:
```suggestion
Fixed '--signing-service' option not allowing to search by href.
```

Reply:
:+1: past tense is preferred here.

Also, everything should be squashed into a single commit.